marklogic-data-hub-central/ui/e2e/cypress/support/components/common/confirmation-modal.tsx

```diff
@@ -33,6 +33,15 @@ class ConfirmationModal {
   getDeleteEntityRelationshipText() {
     return cy.findByLabelText('delete-relationship-text', {timeout: 15000});
   }
+
+  getDeleteEntityRelationshipEditText() {
+    return cy.findByLabelText('delete-relationship-edit-text', {timeout: 15000});
```

Comment:
Seeing failures running this on my local system. Please try increasing this.

Reply:
Increased the timeout value to 25000